pkg/provision/storage/cleanup.go

```diff
@@ -146,7 +145,8 @@ func getSpecCommonPVCCleanupJob(workspace *dw.DevWorkspace, clusterAPI sync.Clus
 							Command: []string{"/bin/sh"},
 							Args: []string{
 								"-c",
-								fmt.Sprintf(cleanupCommandFmt, path.Join(pvcClaimMountPath, workspaceId)),
+								"exit 1",
```

Comment:
This commit/change needs to be removed before the PR can be merged. It's only here to facilitate testing.